ui/app/components/clients/page/overview.hbs

```diff
@@ -3,16 +3,14 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{#if this.byMonthActivityData}}
```

Comment:
This conditional was preventing the `UsageStats` from displaying since the `RunningTotal` component handles the logic of when to display the chart vs. stats.